libyul/backends/evm/EVMDialect.cpp

```diff
@@ -342,6 +360,36 @@ SideEffects EVMDialect::sideEffectsOfInstruction(evmasm::Instruction _instructio
 	};
 }
 
+BuiltinFunctionForEVM const* EVMDialect::verbatimFunction(size_t _arguments, size_t _returnVariables) const
+{
+	pair<size_t, size_t> key{_arguments, _returnVariables};
+	shared_ptr<BuiltinFunctionForEVM const>& f = m_verbatimFunctions[key];
```

Comment:
```suggestion
	shared_ptr<BuiltinFunctionForEVM const>& function = m_verbatimFunctions[key];
```
or even `verbatimFunction`?